scripts/cloud-check-control/cloudAtlasCheckControl.py

```diff
@@ -102,14 +103,34 @@ def __init__(
 
         self.client = None
         self.instanceName = "AtlasChecks"
+        self.localJar = '/tmp/atlas-checks.jar'
```

Comment:
Should this be set to the `jar` param?

Reply:
localJar is where we copy the jar file to if the user gives us a local file or s3 file. It is copied from the remote place to this location for the execution. See "setup_jar" routine.